### PR TITLE
Add some strings

### PR DESCRIPTION
**Pull request by acaliaro**
#83

### CODE REVIEW

File: MVP/Resources/Translations.it.resx [hunk 1]
```diff
@@ -432,4 +426,136 @@
...
+    <value>Semplice ed efficace</value>
+  </data>
+  <data name="onboarding_3_description" xml:space="preserve">
+    <value>Crea direttamente contributi basati su un URL negli appunti. Pre-compilare tutto ciò che possiamo in modo da poter aggiungere rapidamente nuovi contributi.</value>
+  </data>
+  <data name="onboarding_3_title" xml:space="preserve">
+    <value>Aggiunta veloce</value>
+  </data>
+  <data name="sign_in" xml:space="preserve">
+    <value>Sign in</value>
```
Comment (sthewissen) [inline MVP/Resources/Translations.it.resx:553]: This is actually sign in in Italian? :o

Comment (acaliaro) [reply MVP/Resources/Translations.it.resx:448]: We use some English technical words... like Default or Login. Is it possible to use Accedi. Let me know

Comment (sthewissen) [reply MVP/Resources/Translations.it.resx:448]: I'm fine either way, as long as it's clear to the average Italian :) I just thought maybe it was a forgotten text.

Comment (acaliaro) [reply MVP/Resources/Translations.it.resx:448]: No. Some technical words are better in English than Italian

**Pull request merged**